### DISCUSSION TITLE
使用命令行快速开始构建

### DISCUSSION THREAD

**Discussion by Doddddd** [LSPosed/MagiskOnWSALocal#138] [Ideas]
### Is your feature request related to a problem?/你的请求是否与某个问题相关？

否

### Describe the solution you'd like/描述你想要的解决方案

保留上一次构建的配置
./run sh
使用上一次的配置？(y/n)

**Comment by Howard20181** [comment]
你看一下run.sh在构建开始会打印出来构建参数，把命令行参数记一下用build.sh直接构建就好
比如 
`./build.sh --arch x64 --release-type WIF --magisk-ver stable --gapps-brand MindTheGapps --gapps-variant pico --remove-amazon --root-sol magisk`
`./build.sh --release-type WIF`

**Reply by Doddddd** [reply]
> 你看一下run.sh在构建开始会打印出来构建参数，把命令行参数记一下用build.sh直接构建就好 比如 `./build.sh --arch x64 --release-type WIF --magisk-ver stable --gapps-brand MindTheGapps --gapps-variant pico --remove-amazon --root-sol magisk` `./build.sh --release-type WIF`

请问，怎么在这行命令的基础上指定输出目录？

**Reply by Howard20181** [reply]
目前不能指定输出目录

**Reply by Doddddd** [reply]
> 目前不能指定输出目录

好的

**Comment by Doddddd** [comment]
> 你看一下run.sh在构建开始会打印出来构建参数，把命令行参数记一下用build.sh直接构建就好 比如 `./build.sh --arch x64 --release-type WIF --magisk-ver stable --gapps-brand MindTheGapps --gapps-variant pico --remove-amazon --root-sol magisk` `./build.sh --release-type WIF`

谢谢
